runtime/doc/options.txt

```diff
@@ -1068,6 +1068,15 @@ A jump table for the options with a short description can be found at |Q_op|.
 	don't see it when editing.  When you don't change the options, the BOM
 	will be restored when writing the file.
 
+					*'bracketedpaste'* *'bracketedpaste'*
+'bracketedpaste'	boolean	(default on)
```

Comment:
Missing `{Nvim}` tag somewhere (search for other options with this tag).

Missing update to `vim_diff.txt`.